app/models/like.rb

```diff
@@ -0,0 +1,12 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def self.update_post_likes_count(post)
+    if post.likes_counter.nil?
+      post.likes_counter = 1
+    else
+      post.likes_counter += 1
+    end
+  end
```

Comment:
- [ ] Optional suggestion

I recommend using one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_post_likes_count method every time a new post has been saved to the database, then make the update_post_likes_count method 

`after_save :update_post_likes_count`

private

```
def self.update_post_likes_count
  post.increment!(:likes_counter)
end
```